Bottom: 7b720ce9de9142c79338783ff7fba0e74fa22de2
Top:    168b1f25c700317a9751c7e543e0b75af8488abd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 02:29:42 +0100

Remove unused global variable Zoom_Multiplier

Along with a following commented series of numbers which git log
suggests was originally related to pre-defined zoom ratios.


---

diff --git a/src/data.c b/src/data.c
index 3cf6231..ebd9b29 100644
--- a/src/data.c
+++ b/src/data.c
@@ -58,13 +58,3 @@ LibraryType Library;		/* the library */
 bool Bumped;			/* if the undo serial number has changed */
 
 int addedLines;
-
-double Zoom_Multiplier = 0.01;
-
-/*  { 1.5625, 2.2097, 3.125, 4.4194, 6.25, 8.8388,
-      12.5, 17.6777, 25, 35.3553, 50, 70.7106, 100,
-      141.421, 200, 282.848, 400, 565.685, 800, 1131.37,
-      1600, 2262.74, 3200, 4525.48, 6400 };
-*/
-
-
diff --git a/src/data.h b/src/data.h
index 22eea0c..bf8e421 100644
--- a/src/data.h
+++ b/src/data.h
@@ -59,8 +59,6 @@ extern LibraryType Library;
 
 /*extern	DeviceInfoType	PrintingDevice[];*/
 
-extern double Zoom_Multiplier;
-
 extern char *Progname;
 extern char *InputTranslations;
